### ISSUE TITLE
Udp support 

### ISSUE THREAD

**Issue by mokhtarabadi** [genshen/wssocks#12]
Did you have any plan to implement a way to transfer UDP packets through the TCP connection?

For example v2ray can do it

**Comment by genshen** [comment]
sure, UDP support is in plan.

**Comment by mokhtarabadi** [comment]
did you trying anything for this feature, I don't know go too much but I find something like this [https://github.com/bradleyfalzon/udp-to-tcp](https://github.com/bradleyfalzon/udp-to-tcp), can you implement this feature

**Comment by genshen** [comment]
Perhaps UDP within socks5 protocol is more complicated than what bradleyfalzon/udp-to-tcp does.  
I will try to add an initial implementation of UDP support in recent weeks based on the description of [socks5 RFC](https://tools.ietf.org/html/rfc1928#section-7).   
Really thanks for your feedbacks and suggestions.

**Comment by mokhtarabadi** [comment]
Yes for client use socks5 UDP as you mentioned, but I talk about server and how UDP packets must transfered through websocket (which on TCP protocol)

Best example of this is v2ray

**Comment by genshen** [comment]
Yep, you are right😀.  I will check the code implementation of v2ray to see if wssocks can learn something from v2ray.

**Comment by mokhtarabadi** [comment]
thanks for your great project

**Comment by mokhtarabadi** [comment]
Hi, any news about this?
